image_acquisition:
  camera_parameters:
    matrix: [1.289061962331979885e+03,0.000000000000000000e+00,3.644093594041585789e+02,
          0.000000000000000000e+00,1.282227445659755631e+03,2.094685991831462957e+02,
          0.000000000000000000e+00,0.000000000000000000e+00,1.000000000000000000e+00]
    distortion: [1.245543717844518200e-01,-2.597473972245496476e+00,1.995496499458032907e-03,1.289794849173935345e-02,2.000510879977276701e+01]
  camera_device_port: 0
  image_height: 360
  image_width: 480
